main_langage_C_fixme

```diff
@@ -0,0 +1,46 @@
+//-----------------------------------------------------------------------------------//
+// Nom du projet 		  :	    Langage_C_fixme
+// Nom du fichier 		:     main_langage_C_fixme.c
+// Date de création 	:     23.02.2015
+// Date de modification : 	xx.xx.2015
```

Comment:
Yes, you are right. in my text editor is correct, but not here... I will place correct the indentation